src/controllers/menuV2Controller.ts

```
@@ -126,29 +198,173 @@ export const batchUpdateUpdates = async (
+          message: "Items array is required"
+        })
+      }
+
+      let updatedItems = []
+      let Model: any
+
+      // Determine which model to use based on itemType
+      switch (itemType.toLowerCase()) {
+        case 'productos':
+        case 'products':
+          const { Product } = await import("../models/Product")
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)